backend/backend/settings.py

```diff
@@ -183,19 +194,36 @@
         "rest_framework.authentication.TokenAuthentication",
     ],
     "EXCEPTION_HANDLER": "backend.exception_handler.bad_request_logger",
+    "DEFAULT_RENDERER_CLASSES": (
+        "djangorestframework_camel_case.render.CamelCaseJSONRenderer",
+        "djangorestframework_camel_case.render.CamelCaseBrowsableAPIRenderer",
+    ),
+    "DEFAULT_PARSER_CLASSES": (
```

Comment:
@to-sta, this is where I have the parser classes being defined as in the [djangorestframework-camel-case docs](https://github.com/vbabiy/djangorestframework-camel-case), but we're still getting snake_case in responses in the frontend.